icons/ktor.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Ktor</title><path d="M8.001 0 8 .002 4.02 3.984l-.244.243L.002 8 0 8.002h.004l4.223 4.223L8 16l.023-7.973h.174l7.782-.025-.024 7.975h-.154l-7.8.023 3.981 3.982 3.997 3.997v.003l.002-.002L16 24l4.224-4.225L24 16h-.04l.019-.018-4.223-4.224-3.776-3.776-3.965-3.962L8.001.004Zm11.997 11.942L16 8.002v8h8z"/></svg>
```

Comment:
Thank you for the change. Unfortunately, there is still a difference between yours (in black) and the original:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/ba48b86c-287e-44c6-a699-956b3972cab6)

There are also some extra nodes that could be removed. Here is some examples, but there is more:
![image](https://github.com/simple-icons/simple-icons/assets/12817388/984076a4-f171-42e1-b697-eb41fd32c741)
![image](https://github.com/simple-icons/simple-icons/assets/12817388/4bba207b-769b-4d9a-be22-eef66e5e8eef)
![image](https://github.com/simple-icons/simple-icons/assets/12817388/1fdf998a-4a79-4ed6-ae2d-8590138cc2cb)